#! /bin/sh -e

# All lines beginning with `# DPATCH:' are a description of the patch.
# DP: python2.5 doesn't have py3kwarning

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

From b85e9952fd5dbf175b7bdd6ec63521a319ae770d Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Mon, 27 Sep 2010 10:40:03 +0200
Subject: [PATCH 5/9] bsddb: py2.5 doesn't seem to have sys.py3kwarning

---
 Lib/bsddb/__init__.py |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/Lib/bsddb/__init__.py b/Lib/bsddb/__init__.py
index 07c7c9d..0e5169c 100644
--- a/Lib/bsddb/__init__.py
+++ b/Lib/bsddb/__init__.py
@@ -42,12 +42,6 @@ instead.  It mirrors the Oracle Berkeley DB C API.
 import sys
 absolute_import = (sys.version_info[0] >= 3)
 
-if sys.py3kwarning:
-    import warnings
-    warnings.warnpy3k("in 3.x, the bsddb module has been removed; "
-                      "please use the pybsddb project instead",
-                      DeprecationWarning, 2)
-
 try:
     if __name__ == 'bsddb3':
         # import _pybsddb binary as it should be the more recent version from
-- 
1.7.1

